db/seeds.rb

```diff
@@ -120,4 +120,8 @@
 UserRole.create( user_id: 4, role_id: 3)
 UserRole.create( user_id: 5, role_id: 3)
 UserRole.create( user_id: 6, role_id: 3)
-puts "Finished creating UserRoles"
\ No newline at end of file
+puts "Finished creating UserRoles"
+
+puts "Creating Newspapers"
+Newspaper.create( config: 51459807, volume: 1, issue: 1 )
```

Comment:
This isn't the correct volume/issue. Actual configs are in Slack, or message #rails for them. Stop half-ass-ing everything you do.